Sources/SQLKit/SQLQueryEncoder.swift

```diff
@@ -12,6 +13,13 @@ public struct SQLQueryEncoder {
         return encoder.row
     }
 
+    public enum NilEncodingStrategy {
+        /// Standard nil encoding
+        case standard
```

Comment:
Nit/bikeshedding: `default` seems more common in the Vapor circles and appropriate. Also, it might be good to explain here what standard (or default) _means_. Something like: "Skips nilable columns with nil values during encoding."